src/utils/buildImageForm.spec.ts

```diff
@@ -1,11 +1,11 @@
 import FormData from 'form-data';
 
-import { buildImageForm } from './buildImageForm';
+import { buildMultipartBody } from './buildMultipartBody';
 
 describe('buildImageForm', () => {
```

Comment:
See my previous comment about combining this test into a single file with the other multipart body-related tests